newrelic_rpm.gemspec

```diff
@@ -56,6 +56,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest', "#{RUBY_VERSION >= '2.7.0' ? '5.3.3' : '4.7.5'}"
   s.add_development_dependency 'minitest-stub-const', '0.6'
   s.add_development_dependency 'mocha', '~> 1.16'
+  s.add_development_dependency 'mutex_m' # used by Minitest, not included in Rubies above v3.3
+  s.add_development_dependency 'ostruct' # used by Rack, not included in Rubies above v3.4
```

Comment:
thank you for the comments here! That'll be helpful in the future when we eventually forget what these are for